extension/js/common/composer/composer-contacts.ts

```diff
@@ -269,7 +272,7 @@ export class ComposerContacts extends ComposerComponent {
     }
   }
 
-  private renderSearchRes = (input: JQuery<HTMLElement>, contacts: Contact[], query: ProviderContactsQuery) => {
+  private renderSearchRes = (input: JQuery<HTMLElement>, contacts: Contact[], query: ProviderContactsQuery, research: () => Promise<void>) => {
```

Comment:
minor nitpick, `research` sounds like academic activity, maybe let's name it `reSearch`

Reply:
But I'm thinking, would it be possible to trigger re-search without passing a callback? It looks a bit awkward. maybe if you could pass `reSearchDbOnly` into `renderSearchRes`, then you could re-call `searchContacts` directly inside `renderSearchRes` without having to pass a callback?

Reply:
Yes, Agree :)
I'm working on it now

Reply:
I think I can get rid of it at all